src/main/java/com/jcraft/jsch/KeyPairECDSA.java

```diff
@@ -351,7 +351,8 @@ public byte[] getSignature(byte[] data) {
               .asSubclass(SignatureECDSA.class);
       SignatureECDSA ecdsa = c.getDeclaredConstructor().newInstance();
       ecdsa.init();
-      ecdsa.setPrvKey(prv_array);
+      // issue 730: prv_array could be destroyed by ecdsa signing process if its first bit is 1
```

Comment:
Can you fix this to correctly point to issue 739 and not issue 730 (which is unrelated).

Reply:
Also, I would prefer if we included a full link to `https://github.com/mwiede/jsch/issues/739` in the comment text, so that future readers don't have to figure out what `issue 739` means.

Reply:
maked me realize 2 things:
This copy of the private key has to be cleaned... as we don't know if ecdsa signing will clean it...
So I need Util.bzero to be changed to public.
Would you prefer then:
```
      byte[] keyCopy = Arrays.copyOf(prv_array, prv_array.length);
      ecdsa.setPrvKey(keyCopy);
      Util.bzero(keyCopy);
```


I also need to change the caller of setPubKey with the same fix :(

Reply:
I pushed a new secured fixed regarding cleaning the key copy.
The resulting code starts to get ugly because of that unneeded copy...

Reply:
> made me realize 2 things: This copy of the private key has to be cleaned... as we don't know if ecdsa signing will clean it... So I need Util.bzero to be changed to public. Would you prefer then:
> 
> ```
>       byte[] keyCopy = Arrays.copyOf(prv_array, prv_array.length);
>       ecdsa.setPrvKey(keyCopy);
>       Util.bzero(keyCopy);
> ```
> 
> I also need to change the caller of setPubKey with the same fix :(

There is zero reason to zero the public key, as that is not sensitive data.